.travis.yml

```diff
@@ -5,7 +5,7 @@ node_js:
 
 branches:
   only:
-    - gh-pages
+    - master
```

Comment:
This change seems incorrect and unrelated. We don't have a `master` branch; it's `gh-pages` (dating back to when GitHub forced you to use that branch name if you wanted to host content on github.io).

Reply:
I don’t remember why I changed this. Reverted.